### PR TITLE
[GraphX] Improve LiveJournalPageRank example

### PR DESCRIPTION
**Pull request by jackylk**
1. Removed unnecessary import
2. Modified usage print since user must specify the --numEPart parameter as it is required in Analytics.main

### CODE REVIEW

File: examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala [hunk 1]
```diff
@@ -17,11 +17,6 @@
 
 package org.apache.spark.examples.graphx
 
-import org.apache.spark.SparkContext._
```
Comment (srowen) [inline examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala]: Are you sure you can remove this one? it imports implicits right?
This change loses the description of the parameter too.
This is on the border of non-trivial, but if it's merely fixing the usage note, seems OK.

Comment (jackylk) [inline examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala]: Yes, I think it is OK.
The description of the parameter will be output by the Analytics.Main, looks like:
```
JackydeMacBook-Pro:spark jackylee$ bin/run-example graphx.LiveJournalPageRank ../data/Wiki-Vote.txt
Set the number of edge partitions using --numEPart.
Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
JackydeMacBook-Pro:spark jackylee$ 
```

Comment (srowen) [inline examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala]: Yes, but, why remove the description, which already existed in the help message in this file?

Comment (jackylk) [inline examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala]: Yes, you are right. I modified back, just remove the default value description

File: examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala [hunk 2]
```diff
@@ -30,14 +25,14 @@ object LiveJournalPageRank {
...
-        "Usage: LiveJournalPageRank <edge_list_file>\n" +
+        "Usage: LiveJournalPageRank <edge_list_file> --numEPart=<num_edge_partitions>\n" +
           "    [--tol=<tolerance>]\n" +
           "        The tolerance allowed at convergence (smaller => more accurate). Default is " +
           "0.001.\n" +
           "    [--output=<output_file>]\n" +
           "        If specified, the file to write the ranks to.\n" +
           "    [--numEPart=<num_edge_partitions>]\n" +
-          "        The number of partitions for the graph's edge RDD. Default is 4.\n" +
+          "        The number of partitions for the graph's edge RDD.\n" +
```
Comment (srowen) [inline examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala]: Eh, but now you have put the option in twice! why not just move it up and remove the square braces to mark it non-optional? keep everything else the same.

File: bin/pyspark [hunk 3]
```diff
@@ -60,6 +60,9 @@ fi
 #
 # For backwards-compatibility, we retain the old IPYTHON and IPYTHON_OPTS variables.
 
+export PYSPARK_DRIVER_PYTHON=ipython
```
Comment (srowen) [inline bin/pyspark]: OK, but now there is this unrelated change in your PR.

File: examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala [hunk 4]
```diff
@@ -30,14 +25,14 @@ object LiveJournalPageRank {
   def main(args: Array[String]) {
     if (args.length < 1) {
       System.err.println(
-        "Usage: LiveJournalPageRank <edge_list_file>\n" +
+        "Usage: LiveJournalPageRank <edge_list_file> --numEPart=<num_edge_partitions>\n" +
```
Comment (srowen) [inline examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala]: `--numEPart` is still duplicated. It's added to this line but already shown directly below.

Comment (jackylk) [inline examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala]: How about now
